paseos/tests/init_test.py

```diff
@@ -0,0 +1,48 @@
+"""Tests to see if the module can be initialized and set up as planned."""
+
+import sys
+
+sys.path.append("../..")
+
+import paseos
+from paseos import SpacecraftActor
+import pykep as pk
+import numpy as np
+
+
+def test_init():
+    sim = paseos.init_sim()
+    cfg = sim.get_cfg()  # noqa
+    print(cfg)
+
+
+def test_adding_sat():
+    earth = pk.planet.jpl_lp("earth")
+    actor = SpacecraftActor("sat1", [1000000, 0, 0], [0, 8000.0, 0], pk.epoch(0), earth)
+    sim = paseos.init_sim()
+    sim.add_actor(actor)
+
+    # check initial positions
+    r, v = actor.get_position_velocity(pk.epoch(0))
+    assert np.isclose(r[0], 1000000)
+    assert np.isclose(r[1], 0)
+    assert np.isclose(r[2], 0)
+    assert np.isclose(v[0], 0)
+    assert np.isclose(v[1], 8000.0)
+    assert np.isclose(v[2], 0)
+
+    # check positions one second later
+    r, v = actor.get_position_velocity(pk.epoch(1 * pk.SEC2DAY))
+    assert np.isclose(r[0], 999800.6897266058)
```

Comment:
Could you, please, add some comments on the meaning of these magic numbers?

Reply:
r,v are commonly position velocity vector in astrodynamics, added a comment to clarify

Reply:
(the numbers are just values I determined trying out, there is #1 to make more robust tests)